### PR TITLE
Do remaining dispatches in Github Actions too

### PR DESCRIPTION
**Pull request by jordisala1991**

### CODE REVIEW

**Review by VincentLanglet — APPROVED** [review]

File: .github/workflows/dispatch-labels-hooks-topics-settings.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - name: Install PHP with extensions
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: ${{ env.REQUIRED_PHP_EXTENSIONS }}
+                  coverage: pcov
```
Comment (OskarStark) [inline .github/workflows/dispatch-labels-hooks-topics-settings.yaml:52]: Do we need coverage here?

File: .github/workflows/dispatch-labels-hooks-topics-settings.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
+name: Dispatch labels, hooks, topics and settings
+
+on:
+    schedule:
```
Comment (OskarStark) [inline .github/workflows/dispatch-labels-hooks-topics-settings.yaml:4]: Why not run them also on a merge in master?

Comment (jordisala1991) [reply .github/workflows/dispatch-labels-hooks-topics-settings.yaml:4]: My idea is to add a button on actions to manually trigger them. Normally those changes can wait a few hours, it doesnt matter. The problem with doing it on each merge is that we could easily hit rate limits

File: .github/workflows/dispatch-labels-hooks-topics-settings.yaml (hunk 1 above)
Comment (jordisala1991) [reply .github/workflows/dispatch-labels-hooks-topics-settings.yaml:52]: Nope, I will change that

Comment (jordisala1991) [reply .github/workflows/dispatch-labels-hooks-topics-settings.yaml]: Changed

**Review by VincentLanglet — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by jordisala1991** [comment]
Make sure you deploy dev-kit one last time @VincentLanglet

**Comment by VincentLanglet** [comment]
done

**Pull request merged**